src/app/user/artist/component/artist-list/artist-list.component.ts

```diff
@@ -34,11 +37,13 @@ export class ArtistListComponent implements OnInit {
         this.artistList = artists;
         this.config.totalItems = artists.length;
         this.calculateActiveArtTypes();
+        this.filterService = new ArtistFilterService(artists);
```

Comment:
why ?

Reply:
Since the Filter MUST use the original list, DI can't be used here.
Another method would be to DI the service and use Getters and Setters for the list to achieve the same results @gh-assan